src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/DefaultDataCollectionLauncher.cs

```diff
@@ -59,10 +60,15 @@ public override int LaunchDataCollector(IDictionary<string, string> environmentV
         var currentProcessPath = _processHelper.GetCurrentProcessFileName();
 
         // If current process is dotnet/dotnet.exe and you are here, datacollector.exe is present in TestHost folder.
+        // It's unexpected we have DotnetDataCollectionLauncher for .NET Core and we're not shipping arm64 version inside the SDK
+        // because everything there is AnyCPU and will be "Ready to Run" compiled for the specific architecture shipped.
+        // We'll eventually run emulated x64.
```

Comment:
I don't understand this comment. Can you elaborate?

Reply:
It's not clear to me when we reach this point and if it's possible, shouldn't never have dotnet.exe as current process because in that case we use `DotnetDataCollectionLauncher`

https://github.com/microsoft/vstest/blob/1539d8b6c2a6635d9fae371c707b8fcbac7c1be8/src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/DataCollectionLauncherFactory.cs#L44

So I've updated a bit the comment, we won't ship arm64 collector on the sdk so that file won't be there and the default one will be x64.

I don't know if makes sense update the authoring to have also arm version inside the sdk. What do you think?

Reply:
Looks like is used for CodeCoverage, it's not clear, I'll ship for now...but I have to sync with @jakubch1 on it, maybe it's code that we can remove